src/app/ApplicationTemplate.Presentation/ViewModels/MenuViewModel.cs

```diff
@@ -52,6 +59,7 @@ public enum Section
 			.Select(state =>
 			{
 				var vmType = state.GetViewModelType();
+				SelectedIndex = _viewModelsWithBottomMenu.IndexOf(vmType);
```

Comment:
Don't do side effects in select operators.
This should not have been merged.

Reply:
I can check it out, figure out a fix.

Reply:
Probably with an observer.

Reply:
https://github.com/nventive/UnoApplicationTemplate/pull/308